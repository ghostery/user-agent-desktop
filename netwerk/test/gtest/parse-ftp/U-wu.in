ftp> open sunsite.unc.edu
Connected to sunsite.unc.edu.
220 divahouse.metalab.unc.edu FTP server (Version wu-2.6.1(4) Thu Nov 29 23:05:07 EST 2001) ready.
Name (sunsite.unc.edu:cyp): 
331 Guest login ok, send your complete e-mail address as password.
230-
230-             Welcome to ibiblio.org's FTP archives! 
230-  	       formerly known as MetaLab.unc.edu
230-
230-
230-For more information about services offered by ibiblio.org, 
230-browse to http://ibiblio.org/faq
230-
230-You can access this archive via HTTP with the same URL. 
230-
230-example:  ftp://ibiblio.org/pub/Linux/  becomes 
230-         http://ibiblio.org/pub/Linux/, but we prefer you use FTP.
230-
230-You can get tarred directories if you issue a "get dirname.tar"
230-You can also get gzipped or compressed tarred directories by following 
230-the .tar with .gz or .Z, respectively. Please don't issue either of 
230-these commands to get Linux distributions. They are already compressed, 
230-so this only generates unnecessary CPU overhead for us.
230-
230-************************************************************************
230-
230-Please use LSM documentation when submitting to the linux archive.  
230-Anything submitted without an LSM will be rejected!  You'll get an 
230-email form letter about it if we can figure out who you are.
230-
230-To learn more about submitting an LSM, 
230-see: http://www.ibiblio.org/pub/Linux/howtosubmit.html
230-or ftp://www.ibiblio.org/pub/Linux/HOW.TO.SUBMIT
230-
230-***********************************************************************
230-
230-If you mirror a site on ibiblio, please subscribe to our mirror list:
230-http://lists.ibiblio.org/mailman/listinfo/ibiblio-mirrors
230-
230-Have suggestions or questions?  Please mail ftpkeeper@ibiblio.org.
230-
230-
230-Please read the file README
230-  it was last modified on Tue Feb 19 09:46:53 2002 - 140 days ago
230 Guest login ok, access restrictions apply.
Remote system type is UNIX.
Using binary mode to transfer files.
ftp> syst
215 UNIX Type: L8
ftp> ls
200 PORT command successful.
150 Opening ASCII mode data connection for /bin/ls.
total 78744
-r--r--r--    1 root     other    80507411 Jul  9 06:09 IAFA-LISTINGS
-rw-r--r--    1 root     root         1393 Feb 19 14:46 README
dr-xr-xr-x    2 root     other        4096 Mar 27  2001 bin
dr-xr-xr-x    2 root     other        4096 Jul 16  1997 dev
dr-xr-xr-x    2 root     other        4096 May 30 14:35 etc
drwxrwxrwx   18 ftp      20           4096 Jun 10 13:05 incoming
drwxr-xr-x    2 root     root         4096 Mar 27  2001 lib
lrwxrwxrwx    1 root     other          13 Mar 16  2001 ls-lR -> IAFA-LISTINGS
dr-xr-xr-x   18 root     root         4096 May 31 19:46 pub
dr-xr-xr-x    3 root     other        4096 Jun 26  2000 unc
dr-xr-xr-x    5 root     other        4096 Jul 16  1997 usr
226 Transfer complete.
ftp> close
221-You have transferred 0 bytes in 0 files.
221-Total traffic for this session was 2777 bytes in 1 transfers.
221-Thank you for using the FTP service on divahouse.metalab.unc.edu.
221 Goodbye.
ftp> 