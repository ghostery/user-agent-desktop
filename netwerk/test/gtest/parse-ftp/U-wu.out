<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

07-09-2002  06:09:00             80507411 IAFA-LISTINGS
02-19-2002  14:46:00                 1393 README
03-27-2001  00:00:00       <DIR>          bin
07-16-1997  00:00:00       <DIR>          dev
05-30-2002  14:35:00       <DIR>          etc
06-10-2002  13:05:00       <DIR>          incoming
03-27-2001  00:00:00       <DIR>          lib
03-16-2001  00:00:00       <JUNCTION>     ls-lR -> IAFA-LISTINGS
05-31-2002  19:46:00       <DIR>          pub
06-26-2000  00:00:00       <DIR>          unc
07-16-1997  00:00:00       <DIR>          usr
